src/chromium/crPage.ts

```diff
@@ -364,10 +366,14 @@ class FrameSession {
     ];
   }
 
-  async _initialize() {
+  async _initialize(isHeadful: boolean) {
```

Comment:
nit: can you give the parameter a name that would make more sense for headful oopif or pass actual isHeadful value in addFrameSession? It took me some time to figure out that it's not a bug in addFrameSession.

Reply:
Done